### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by zhanxw** [jiwoongbio/NanoPrism#1]
1. "-t" options
2. example on normalization 
Thanks.

**Comment by zhanxw** [comment]
For the `-t` option, can users input multiple taxonomy IDs?

**Comment by jiwoongbio** [comment]
Yes. You can specify comma-separated taxonomy IDs or multiple -t options like -t 287 -t 562

**Comment by zhanxw** [comment]
Is the output result already normalized?

e.g. ERR3152364.NanoPrism.abundance.txt

```
K00001	0.42701249216754
K00003	0.107892487472779
K00004	0.528139684414812
K00005	0.172696194384774
K00008	0.322533489754937
K00009	0.0926147658197949
...
```

**Comment by jiwoongbio** [comment]
Yes. The values are normalized by dividing by the mean value of K02950, K02874, K02946, K02948, K02867, K02952, K02886, K02988, K02992, K02965.
